lib/ability.rb

```diff
@@ -26,7 +26,11 @@ def initialize(user, resource, controller = nil)
         end
         unless user.account_manager?
           cannot :manage, User unless resource.is_a?(Facility) && resource.single_facility?
-          cannot([:edit, :update], User) { |target_user| !target_user.admin_editable? }
+          if SettingsHelper.feature_on?(:create_users)
+            cannot([:edit, :update], User) { |target_user| !target_user.admin_editable? }
+          else
+            cannot([:edit, :update], User)
```

Comment:
If we change this line to `cannot :manage, User` we will also remove indexing and viewing--the Users tab literally will not appear in the admin view.